bin/torii/src/main.rs

```diff
@@ -160,7 +160,8 @@ async fn main() -> anyhow::Result<()> {
     // Get world address
     let world = WorldContractReader::new(args.world_address, &provider);
 
-    let db = Sql::new(pool.clone(), args.world_address).await?;
+    let class_hash = world.base().call().await?;
```

Comment:
This is the class hash of the `base` contract, not the `world`. I don't think cainome exposes something for that, we should add it to be able to to something lime `world.class_hash()`.

I can add it if you need to, or you may use starknet-rs in the meantime to use the `getClassHashAt`.